pkgs/top-level/all-packages.nix

```diff
@@ -19501,11 +19501,8 @@ with pkgs;
   openvdb = callPackage ../development/libraries/openvdb {};
 
   inherit (callPackages ../development/libraries/libressl { })
-    libressl_3_2
```

Comment:
We should add a throw for this attribute over in `aliases.nix`.

Reply:
Pushed this. Apologies @fpletz , just wanted to get a bit of a move on with this.